File: app/src/main/java/com/starostinvlad/fan/GsonModels/Serial.java
package com.starostinvlad.fan.GsonModels;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Serial implements Serializable {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("closed")
    @Expose
    private Boolean closed;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Boolean getClosed() {
        return closed;
    }

    public void setClosed(Boolean closed) {
        this.closed = closed;
    }

}

File: app/src/main/java/com/starostinvlad/fan/GsonModels/Serial.kt
package com.starostinvlad.fan.GsonModels

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable
import kotlin.Throws

class Serial : Serializable {
    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("closed")
    @Expose
    var closed: Boolean? = null
}